lib/Gedmo/Sortable/Comparable.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Gedmo\Sortable;
+
+/**
+ * This interface is not necessary but can be implemented for
+ * Entities which has custom compare methods
+ * Comparable
+ *
+ * @link https://wiki.php.net/rfc/comparable
+ * @author Raine Ng <yellow1912@gmail.com>
+ * @license MIT License (http://www.opensource.org/licenses/mit-license.php)
+ */
+interface Comparable
+{
+    public function compareTo($other);
```

Comment:
please specify that it should return boolean if it matches. because in most languages compareTo returns 0 if equals -1 if less and 1 if greater..